R/contingencytables.R

```diff
@@ -413,14 +418,14 @@ ContingencyTablesInternal <- function(jaspResults, dataset, options, ...) {
       pr.type   <- "number"
     }
 
-                                         table$addColumnInfo(name = paste0(column.name,"[counts]"),                  title = myTitle, type = pr.type,  format = pr.format, overtitle = overTitle)
-    if (options$countsExpected)          table$addColumnInfo(name = paste0(column.name,"[expected]"),                title = myTitle, type = "number", format = "sf:4;dp:2")
-    if (options$percentagesRow)          table$addColumnInfo(name = paste0(column.name,"[row.proportions]"),         title = myTitle, type = "number", format = "dp:1;pc")
-    if (options$percentagesColumn)       table$addColumnInfo(name = paste0(column.name,"[col.proportions]"),         title = myTitle, type = "number", format = "dp:1;pc")
-    if (options$percentagesTotal)        table$addColumnInfo(name = paste0(column.name,"[total.proportions]"),             title = myTitle, type = "number", format = "dp:1;pc")
-    if (options$residualsUnstandardized) table$addColumnInfo(name = paste0(column.name,"[unstandardized.residuals]"),title = myTitle, type = "number", format = "sf:4;dp:2")
-    if (options$residualsPearson)        table$addColumnInfo(name = paste0(column.name,"[pearson.residuals]"),       title = myTitle, type = "number", format = "sf:4;dp:2")
-    if (options$residualsStandardized)   table$addColumnInfo(name = paste0(column.name,"[standardized.residuals]"),  title = myTitle, type = "number", format = "sf:4;dp:2")
+    if (options$countsObserved)          table$addColumnInfo(name = paste0(column.name,"[counts]"),                  title = myTitle, type = pr.type,  format = pr.format,   overtitle = overTitle)
+    if (options$countsExpected)          table$addColumnInfo(name = paste0(column.name,"[expected]"),                title = myTitle, type = "number", format = "sf:4;dp:2", overtitle = overTitle)
```

Comment:
Do we still use this `format` argument? Doesn't it mess with the decimals that the user globally specifies?